### PR TITLE
docs: new introduction of GreptimeDB

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

ref to https://github.com/GreptimeTeam/greptimedb/pull/3099

## Checklist

- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.5/zh/index.md [hunk 1]
```diff
@@ -4,24 +4,26 @@
     <img src="/logo-greptimedb.png" alt="GreptimeDB Logo" width="400px">
 </p>
 
-GreptimeDB 是开源的时序数据库，注重可扩展性、分析能力和效率。GreptimeDB 可以在云时代的基础设施上工作，为用户提供良好的灵活性和商品存储服务。
+GreptimeDB 是开源的时序数据库，注重效率、可扩展性和分析能力。GreptimeDB 可以在云时代的基础设施上工作，为用户提供良好的灵活性和商品存储服务。
```
Comment (killme2008) [inline docs/v0.5/zh/index.md:7]:
```suggestion
GreptimeDB 是开源的时序数据库，注重效率、可扩展性和分析能力。GreptimeDB 可以在云时代的基础设施上工作，得益于良好的灵活性和廉价存储服务。
```

File: docs/v0.5/zh/index.md [hunk 2]
```diff
@@ -4,24 +4,26 @@
...
 
-- 可扩展到高可用的分布式集群的[单机版 binary](https://github.com/GreptimeTeam/greptimedb/releases)，为集群用户提供透明的体验
-- 为处理时序数据而优化的列式布局；经过压缩、整理，并存储在各种存储后端
-- 灵活的索引，解决高基数的问题
-- 利用弹性计算资源执行分布式并行查询
-- 用于高级分析场景的本地 SQL 和 Python 脚本
-- 广泛采用的数据库协议和 API，支持本地 PromQL
-- 适用于广泛的工作负载可扩展的表引擎架构
-- Schemaless 写入，自动为数据创建表格
+- 为处理时序数据而优化的列式布局；经过压缩、整理，并存储在各种存储后端，特别是具有 50 倍成本效率的云对象存储。
```
Comment (killme2008) [inline docs/v0.5/zh/index.md:14]:
```suggestion
- 为处理时序数据而优化的列式布局；经过压缩、整理，并存储在各种存储后端，尤其是可以节省 50 倍成本效率的云对象存储。
```

File: docs/v0.5/zh/index.md [hunk 3]
```diff
@@ -4,24 +4,26 @@
...
+- 完全开源的分布式集群架构，利用云原生弹性计算资源的强大功能。
+- 从边缘节点的独立二进制文件到云端强大、高可用的分布式集群的无缝扩展，为开发人员和管理员提供透明的体验。
```
Comment (killme2008) [inline docs/v0.5/zh/index.md:16]:
```suggestion
- 从边缘节点的单机部署到云端强大、高可用的分布式集群的无缝扩展，为开发人员和管理员提供透明的体验。
```

File: docs/v0.5/zh/index.md [hunk 4]
```diff
@@ -4,24 +4,26 @@
...
+- 原生 SQL、PromQL 查询及 Python 脚本，以便于复杂的分析任务。
```
Comment (killme2008) [inline docs/v0.5/zh/index.md:17]:
```suggestion
- 原生 SQL 和 PromQL 的查询支持，以及便于复杂的分析任务的 Python 脚本。
```

File: docs/v0.5/zh/index.md [hunk 5]
```diff
@@ -4,24 +4,26 @@
...
+- 灵活的索引功能和分布式、并行处理查询引擎，解决高基数的问题。
+- 广泛采用的数据库协议和 API，包括 MySQL、PostgreSQL 和 Prometheus 远程存储等。
```
Comment (killme2008) [inline docs/v0.5/zh/index.md:19]:
```suggestion
- 适配广泛采用的数据库协议和 API，包括 MySQL、PostgreSQL 和 Prometheus 远程存储等。
```

File: docs/v0.5/zh/index.md (hunk 1 above)
Comment (sunng87) [reply docs/v0.5/zh/index.md]: 廉价 -> 高性价比

File: docs/v0.5/zh/index.md [hunk 6]
```diff
@@ -4,24 +4,26 @@
     <img src="/logo-greptimedb.png" alt="GreptimeDB Logo" width="400px">
 </p>
 
-GreptimeDB 是开源的时序数据库，注重可扩展性、分析能力和效率。GreptimeDB 可以在云时代的基础设施上工作，为用户提供良好的灵活性和商品存储服务。
+GreptimeDB 是开源的时序数据库，注重效率、可扩展性和分析能力。GreptimeDB 可以在云时代的基础设施上工作，使用户受益于良好的灵活性和廉价存储服务。
```
Comment (sunng87) [inline docs/v0.5/zh/index.md:7]:
```suggestion
GreptimeDB 是开源的时序数据库，注重效率、可扩展性和分析能力。GreptimeDB 可以在云时代的基础设施上工作，使用户受益于良好的灵活性和高性价比的存储。
```

**Review by sunng87 — APPROVED** [review]

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**